test/resources/sr_rules.json

```diff
@@ -172,7 +172,7 @@
           "group": ["A", "B"],
           "schema": [
             {
-              "uri": "https://eu.com/claims/DriversLicense"
+              "uri": "https://www.w3.org/2018/credentials/v1"
```

Comment:
we had to move these to TypeScript files from json files.

ps. all json example should be moved to ts.